mlflow/gateway/providers/clarifai.py

```diff
@@ -0,0 +1,282 @@
+        #                 "app_id": "main",
+        #                 "model_version": {
+        #                     "id": "9fe2c8962c104327bc87b8f8104b161a",
+        #                     "created_at": "2022-11-14T15:43:30.757520Z",
+        #                     "status": {
+        #                         "code": 21100,
+        #                         "description": "Model is trained and ready"
+        #                     },
+        #                     "train_stats": {},
+        #                     "completed_at": "2022-11-14T15:43:30.757520Z",
+        #                     "visibility": {
+        #                         "gettable": 50
+        #                     },
+        #                     "app_id": "main",
+        #                     "user_id": "clarifai",
+        #                     "metadata": {}
+        #                 },
+        #                 "user_id": "clarifai",
+        #                 "model_type_id": "multimodal-embedder",
+        #                 "visibility": {
+        #                     "gettable": 50
+        #                 },
+        #                 "toolkits": [],
+        #                 "use_cases": [],
+        #                 "languages": [],
+        #                 "languages_full": [],
+        #                 "check_consents": [],
+        #                 "workflow_recommended": false
+        #             },
+        #             "input": {
+        #                 "id": "c784baf68b204681baf955e7aa1771ba",
+        #                 "data": {
+        #                     "text": {
+        #                         "raw": "Hello World",
+        #                         "url": "https://samples.clarifai.com/placeholder.gif"
+        #                     }
+        #                 }
+        #             },
+        #             "data": {
+        #                 "embeddings": [
+        #                     {
+        #                         "vector": [
+        #                             -0.004586036,
+        #                             0.009094779,
+        #                             -0.010943364,
+        #                             -0.011651881,
+        #                             -0.008251,
+        #                             ...,
+        #                             -0.025429312
+        #                         ],
+        #                         "num_dimensions": 512
+        #                     }
+        #                 ]
+        #             }
+        #         }
+        #     ]
+        # }
+        # ```
+
+        embedding_vectors = [
+            resp["outputs"][i]["data"]["embeddings"][j]["vector"]
+            for i in range(len(resp["outputs"]))
+            for j in range(len(resp["outputs"][i]["data"]["embeddings"]))
+        ]
+        embedding_vectors = [
+            list(map(float, embedding_vectors[i])) for i in range(len(embedding_vectors))
+        ]
```

Comment:
```suggestion
        embedding_vectors = [
            list(map(float, vector["vector"]))
            for output in resp["outputs"]
            for vector in output["data"]["embeddings"]
        ]
```
I think moving to a list comprehension is a bit more readable and combining the mapping to `List[float]` within a single comprehension will be a bit more performant.